### PR TITLE
chore(text-inputs): add paparazzi tests

### LINKED ISSUE #97: test(text-input): Add paparazzi tests
**Describe the solution you'd like**
Write tests with paparazzi for text-input component for all variants & themes. 

**Additional context**
FYI Buttons already have ones.

### PR DESCRIPTION
**Pull request by FDELAHA24**
## Changes description 🧑‍💻

Add paparazzi tests

## Context 🤔

Closes #97 

## Checklist ✅

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check compose contract convention. It must follow conventions described [here](/CONVENTIONS.md).
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [ ] I have tested on a phone device/emulator.
- [ ] I have tested on a tablet device/emulator.
- [ ] I have tested on a large screen device/emulator.
- [ ] If it includes design changes, please ask for a review `design-system-core-team-design` GitHub team.

### CODE REVIEW

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/4402fb34-e89a-47dc-be80-98882490433f/artifact/ce6f1efb9f664cf7/p/74150b7fe69db08cb8ffd146c508aea6

File: text-inputs/src/test/snapshots/images/com.decathlon.vitamin.compose.textinputs_VitaminTextInputsPrimaryTest_normal[Filled,Dark].png
Comment (ManonPolle) [inline text-inputs/src/test/snapshots/images/com.decathlon.vitamin.compose.textinputs_VitaminTextInputsPrimaryTest_normal[Filled,Dark].png:1]: Disabled placeholder color is to be check here. Barely visible in Light mode, seems worse in dark;

**Review by ManonPolle — APPROVED** [review]

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/d5bf38c8-ab54-498f-9fa5-b160858cd2c8/artifact/7a9cb04dd1be5065/p/38bfca322a72af5057c20522f13855bd

**Review by SimonLeclercq — COMMENTED** [review]
Does Design team have something to check? :)

**Review by SimonLeclercq — APPROVED** [review]

**Pull request merged**